etc/nginx/nginx.conf.template

```diff
@@ -52,6 +52,8 @@ http {
          auth_ldap_servers ldapserver;
          uwsgi_pass django;
          include /etc/nginx/uwsgi_params;
+         # http://nginx.org/en/docs/http/ngx_http_uwsgi_module.html#uwsgi_read_timeout
+         uwsgi_read_timeout 60s;
```

Comment:
According to the linked docs, this is already the default.

Reply:
yea, i made it explicit because when i increased it i made the box go....to a dark place :)

Reply:
ah. makes sense :D